### ISSUE TITLE
Development server won't launch using `rake hydra:server`

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/mahonia#77]

**Comment by no-reply** [comment]
This is also an issue with `rails console`

**Comment by mark-dce** [comment]
@bess FYI - this ticket was opened on Thursday 10/21 :(

**Comment by mark-dce** [comment]
```
MARKs-MacBook-Pro-2:mahonia mark$ git bisect bad
2312331b66fbc5de8078bf176dd5f50dce6d5ce1 is the first bad commit
commit 2312331b66fbc5de8078bf176dd5f50dce6d5ce1
Author: Tom Johnson <tom@dp.la>
Date:   Tue Oct 10 09:16:32 2017 -0700

    Turn on eager load accross environments
    
    We have run into problems in the past with different autoload behavior in
    development and test. We are aiming to have a practice of treating code loading
    similarly in dev/test as in production to reduce surprises. Class caching
    remains off in development for inline code reloading in development.

:040000 040000 b316a9d079437d4dd7f22c90bbf6c4d032ae11d0 0af9014727667e44fd4e7b7215f5caeced920da6 M	config
MARKs-MacBook-Pro-2:mahonia mark$
```

**Comment by no-reply** [comment]
This was fixed by 62909a9e054, but the underlying hyrax issue remains.

Ideally, we could revert that commit soon after an upstream fix.
